x/auth/tx/service.go

```diff
@@ -252,6 +252,47 @@ func (s txServer) BroadcastTx(ctx context.Context, req *txtypes.BroadcastTxReque
 	return client.TxServiceBroadcast(ctx, s.clientCtx, req)
 }
 
+func (s txServer) TxEncode(ctx context.Context, req *txtypes.TxEncodeRequest) (*txtypes.TxEncodeResponse, error) {
```

Comment:
Can we update the docs here too please: https://github.com/cosmos/cosmos-sdk/blob/main/x/auth/README.md#grpc

Reply:
I don't think this should go in auth grpc section 
@amaurym should we have separate grpc client section docs  for these endpoint tests ?

Reply:
Right! My bad. We should definitely have a section that explains these. Let me check where would fit best.